content/docs/studio/user-guide/prepare-your-repositories.md

```diff
@@ -37,7 +37,8 @@ Datasets, metrics, and hyperparameters can be added to a project in two ways:
    about the features and benefits of DVC [here](/doc/start).
 
    Refer to the [DVC documentation](https://dvc.org/doc) to initialize a DVC
-   repository.
+   repository, or Iterative Studio can
+   [initialize DVC for you](/doc/studio/user-guide/model-registry/add-a-model).
```

Comment:
Maybe an admon would be better here, something like:

```suggestion
   repository.

   <admon type="tip">

   Iterative Studio Model Registries can create DVC repos for you when
   [importing model files](/doc/studio/user-guide/model-registry/add-a-model)
   from remote/external locations.

   </admon>
```

Reply:
Clarified now (suggestion updated, but feel free to further edit it).

Reply:
Updated.